pkg/web/queries.go

```diff
@@ -157,6 +157,55 @@ func (c *Controller) createStoredQuery(ctx *gin.Context) {
 	})
 }
 
+func (c *Controller) updateOrder(ctx *gin.Context) {
+	updateSQL := `UPDATE stored_queries SET num = ( CASE`
```

Comment:
This generated case feels a bit tricky. I would prefer a transaction running a batch update in it.